tests/fixtures/__recordings__/metric-MergedPRsMetric_4095279630/no-problems-for-test-repo-when-no-merged-PRs-are-found_212315159/recording.har

```diff
@@ -82,7 +82,7 @@
             },
             {
               "name": "date",
-              "value": "Thu, 06 May 2021 17:48:09 GMT"
+              "value": "Mon, 10 May 2021 16:12:36 GMT"
```

Comment:
Could you explain what happened here?

Reply:
when the recordings are re-recorded, some of this metadata changes.
I was toying with adding a normalization step to make values like this (and other irrelevant headers like x-api-requests-remaining or whatever it's called) set to a constant value, but didn' get to it yet. I agree it's a little confusing, but I think it's ok for the time being? Ideally there won't be a lot of refreshing after activity on this repo settles down